components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -138,6 +141,31 @@ interface BraveWalletProvider {
   AddSuggestToken(BlockchainToken token) => (bool approved, ProviderError error, string error_message);
 };
 
+interface SolanaProvider {
+  // Optional {onlyIfTrusted: true}, when that flag is enable, we will only
+  // connect and emit a connect event if the application is trusted.
```

Comment:
what does it mean for the application to be trusted?

Reply:
It's like ethereum's connected. For our implementation in browser code, they are both permission stored in content settings

Reply:
It's from https://docs.phantom.app/integrating/establishing-a-connection#eagerly-connecting